test/jdk/java/nio/channels/DatagramChannel/SelectWhenRefused.java

```diff
@@ -53,6 +55,7 @@ public static void main(String[] args) throws IOException {
             sendDatagram(dc, refuser);
             int n = sel.select(2000);
             if (n > 0) {
```

Comment:
This has been seen as the location of the intermittent failures, so maybe adding a receive here also to determine the src of the unexpected datagram would be informative